pimd/pim_upstream.c

```diff
@@ -705,14 +705,14 @@ void pim_upstream_switch(struct pim_instance *pim, struct pim_upstream *up,
 
 	if (pim_addr_is_any(up->upstream_addr)) {
 		if (PIM_DEBUG_PIM_EVENTS)
-			zlog_debug("%s: RPF not configured for %s", __func__,
+			zlog_debug("%s: RP not configured for %s", __func__,
 				   up->sg_str);
 		return;
 	}
 
 	if (!up->rpf.source_nexthop.interface)  {
 		if (PIM_DEBUG_PIM_EVENTS)
-			zlog_debug("%s: RP not reachable for %s", __func__,
+			zlog_debug("%s: RPF not reachable for %s", __func__,
```

Comment:
Again if the interface is NULL then the RPF failed.

Reply:
Yes, RPF is about the nexthop,
and there are other similar logs, like in function 'pim_upstream_send_join'.